src/documentation/ntfs_to_netex_france_specs.md

```diff
@@ -283,3 +284,90 @@ Line/@id | lines.txt | line_id | see [id formatting](#id-of-objects)
 Line/PublicCode | lines.txt | line_code | If the code line_code is empty, this node is not created.
 Line/Name | lines.txt | line_name | 
 
+## calendriers.xml
+
+### Top level structure
+
+Example: 
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<GeneralFrame 
+		id="FR:GeneralFrame:NETEX_CALENDRIER:<stop_provider_code>" 
+		version="any">
+	<ValidBetween>
+		<FromDate>2020-01-01T00:00:00Z</FromDate>
+		<ToDate>2020-12-30T23:59:59Z</ToDate>
+	</ValidBetween>
+	<members>
+		<!-- One DayType for each 'service_id' -->
+		<DayType />
+		<!-- One OperatingPeriod for each line of 'calendar.txt' -->
+		<OperatingPeriod />
+		<!-- One DayTypeAssignment for each line of 'calendar.txt' to link
+		     a DayType and an OperatingPeriod -->
+		<DayTypeAssignment />
+		<!-- One DayTypeAssignment for each line of 'calendar_dates.txt' to link
+		     a DayType and specific date exception (include or exclude) -->
+		<DayTypeAssignment />
+	</members>
+</GeneralFrame>
+```
+
+### ValidBetween
+
+Netex field | NTFS file | NTFS field | Note
+ValidBetween/FromDate | feed_infos.txt | feed_start_date | use ISO-8601 date-time format, using `T00:00:00Z` for the time
+ValidBetween/ToDate | feed_infos.txt | feed_end_date | use ISO-8601 date-time format, using `T23:59:59Z` for the time
+
+### DayType
+
+Netex field | NTFS file | NTFS field | Note
+--- | --- | --- | ---
+DayType/@id | calendar.txt or calendar_dates.txt | service_id | see [id formatting](#id-of-objects)
+DayType/@version | | | fixed value `any`
+DayType/properties/PropertyOfDay[]/DaysOfWeek | calendar.txt | | see (1)
+
+**(1) Week pattern**
+
+One `PropertyOfDay` for each weekday of `calendar.txt`. For `service_id` which
+exists only in `calendar_dates.txt`, no `PropertyOfDay` is created (in NeTEx
+France profile, no `PropertyOfDay` means all the days of the week are active).
+
+`DaysOfWeek` can take one of the following values: `Monday`, `Tuesday`,
+`Wednesday`, `Thursday`, `Friday`, `Saturday`, or `Sunday`.
+
+### OperatingPeriod
+
```

Comment:
`An OperatingPeriod is only created for service_id used in calendars.txt.`

Reply:
Done in 138790f96d3559a33e13eb909d88da9ea499f553.